src/frontend/planner_test/tests/testdata/insert.yaml

```diff
@@ -142,3 +142,39 @@
           | └─BatchScan { table: t2, columns: [t2.c, t2.d], distribution: SomeShard }
           └─BatchExchange { order: [], dist: HashShard(t3.f) }
             └─BatchScan { table: t3, columns: [t3.e, t3.f], distribution: SomeShard }
+- name: basic insert with returning
+  sql: |
+    create table t (a int, b int);
+    insert into t values (0,1), (1,2) returning *, a, a+b;
+  logical_plan: |
+    LogicalProject { exprs: [*VALUES*_0.column_0, *VALUES*_0.column_1, *VALUES*_0.column_0, (*VALUES*_0.column_0 + *VALUES*_0.column_1)] }
+    └─LogicalInsert { table: t }
```

Comment:
It'll be better if we can add a new field of `returning` in `fmt::Display`. Only show it when `returning` is true.

Reply:
One problem is that current we do not hold returning projection info in LogicalInsert/Delete/Update struct but just a bool indicating if there were returning. Do you think it will be better if we print ```LogicalInsert { table: t, returning: true }```?

Reply:
`LogicalInsert { table: t, returning: true }` looks clear 🤣